### PR TITLE
Add try catch to i18n.get

### LINKED ISSUE #2165: Difference in translation arguments can break the client
When there are differences in the translation arguments or an argument itself is translated such as number to nummer it can lead to crashing the Application thread of the client which results in a non-operational client.

Ideally this situation should just catch the error and return the english rather than crashing.

### PR DESCRIPTION
**Pull request by Sheikah45**
Fixes #2165

### CODE REVIEW

File: src/main/java/com/faforever/client/i18n/I18n.java [hunk 1]
```diff
@@ -93,15 +95,25 @@ public String get(String key, Object... args) {
...
     return getWithDefault(userSpecificLocale, defaultMessage, key, args);
   }
 
   public String getWithDefault(Locale locale, String defaultMessage, String key, Object... args) {
-    return messageSource.getMessage(key, args, defaultMessage, locale);
+    try {
+      return messageSource.getMessage(key, args, defaultMessage, locale);
+    } catch (Exception e) {
+      log.error(String.format("Could not load message %s with locale %s defaulting to US english", key, locale.toString()), e);
+      return messageSource.getMessage(key, args, Locale.US);
```
Comment (Brutus5000) [inline src/main/java/com/faforever/client/i18n/I18n.java]: The default message is missing here

File: src/main/java/com/faforever/client/i18n/I18n.java [hunk 2]
```diff
@@ -93,15 +95,25 @@ public String get(String key, Object... args) {
   }
 
   public String get(Locale locale, String key, Object... args) {
-    return messageSource.getMessage(key, args, locale);
+    try {
+      return messageSource.getMessage(key, args, locale);
+    } catch (Exception e) {
+      log.error(String.format("Could not load message %s with locale %s defaulting to english", key, locale.toString()), e);
```
Comment (Brutus5000) [inline src/main/java/com/faforever/client/i18n/I18n.java]: Slf4j logger never need String.format or MessageFormat. It works out of the box with `{}` as the placeholder

File: src/main/java/com/faforever/client/i18n/I18n.java [hunk 3]
```diff
@@ -93,15 +95,25 @@ public String get(String key, Object... args) {
...
   public String getWithDefault(String defaultMessage, String key, Object... args) {
     return getWithDefault(userSpecificLocale, defaultMessage, key, args);
   }
 
   public String getWithDefault(Locale locale, String defaultMessage, String key, Object... args) {
-    return messageSource.getMessage(key, args, defaultMessage, locale);
+    try {
+      return messageSource.getMessage(key, args, defaultMessage, locale);
+    } catch (Exception e) {
+      log.error(String.format("Could not load message %s with locale %s defaulting to US english", key, locale.toString()), e);
```
Comment (Brutus5000) [inline src/main/java/com/faforever/client/i18n/I18n.java]: Same as above

**Review by Brutus5000 — CHANGES_REQUESTED** [review]

File: src/main/java/com/faforever/client/i18n/I18n.java (hunk 2 above)
Comment (Sheikah45) [reply src/main/java/com/faforever/client/i18n/I18n.java]: It cannot take the variable argument to the string and log the throwable but I can just do string concatenation

**Review by Brutus5000 — APPROVED** [review]

**Pull request merged**